ui/app/manager/src/pages/page-configuration.ts

```diff
@@ -299,33 +390,52 @@ export class PageConfiguration extends Page<AppStateKeyed> {
 
     // FETCH METHODS
 
+    protected async uploadCustomMap(e: CustomEvent) {
+        const file = e.detail.value[0] as File;
+        this.tilesForUpload = file;
+        this.requestUpdate()
```

Comment:
You can add the `@state` decorator to the `tilesForUpload: File` field, so you don't have to manually request a state update anymore.